.changeset/dry-ghosts-like.md

```
@@ -0,0 +1,23 @@
+---
+'@mastra/platform-workspace': minor
+---
+
+Added asynchronous sandbox template builds, reusable template handles, and lazy repository templates for Platform workspaces. `PlatformSandbox` can now resolve a providerless definition only when a fresh sandbox is required, wait for its build, and fall back to the provider default when resolution or building fails. `createRepoTemplate()` produces credential-free, commit-pinned definitions for public GitHub repositories.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)